tests/transactional/enable_selinux.pm

```diff
@@ -49,14 +68,21 @@ sub run {
         }
     }
 
-    assert_script_run('selinuxenabled');
-    validate_script_output("getenforce", sub { m/Enforcing/ });
-    validate_script_output("sestatus", sub { m/Current mode:.*enforcing/ });
-    validate_script_output("sestatus", sub { m/Mode from config file:.*enforcing/ });
-    record_info('SELinux', script_output('sestatus'));
-    record_info('Audit report', script_output('aureport'));
-    record_info('Audit denials', script_output('aureport -a', proceed_on_failure => 1));
-    upload_logs($audit_log);
```

Comment:
Here used to be audit log uploaded. You postponed it to later. If `check_enforcing` fails, you're without this log. I'm not sure, but IMHO that would make debugging harder. Maybe move uploading logs to custom function, which would be called in [`post_fail_hook`](https://open.qa/docs/#_how_to_write_tests) and `post_run_hook`?